### ISSUE TITLE
Organisation iHack Sherbrooke 2018

### ISSUE THREAD

**Issue by Sytten** [JDIS/Organisation#19]
Date 10 juin 2018

- [ ] Mise en contact avec l'équipe du Hackfest
- [ ] Réservation du local
- [ ] Achat des prix de présence
- [ ] Achat de la pizza
- [ ] Achat de la bière

**Comment by Res260** [comment]
- Jvais avoir besoin du contact du Hackfest
- Local: Espace Inc. fait encore la job? Ça accueille combien de monde? Combien on était l'année passée?
- Prix de présence: Quel était le budget l'année passée? Est-ce qu'on se donne un budget semblable cette année?
- Pizza: On faisait affaire avec qui l'année passée?
- Bière: Même question.

**Comment by Sytten** [comment]
- Contact: Je te l'envois en PM
- Local: espace inc c'est vraiment nice, on devrait pouvoir l'avoir gratuit ou vraiment pas chère. Ça peut accueillir 60-70 personnes facilement et on était 30-40 l'an passé (notre plus grosse année à date). Normalement on arrive à avoir une commandite de 500$ de sherbrooke innopole pour cet événement. Je t'envois le contact du gars qui deal avec ça.
- Prix: mettons 100$, souvent on fait faire des autocollants pour le monde c'est nice.
- Pizza: surement dominos comme toujours, @maximevaillancourt tu confirme?
- Bière: une année on a acheté un keg, mais yen ai resté. Faque maintenant on achète des caisses. Mais il en reste 3 de l'an passé à mon appartement.

**Comment by Res260** [comment]
Email envoyé à l'équipe du HackFest

**Comment by Res260** [comment]
Niveau tables, c'est à l'espace Inc j'imagine? Pour le matériel (switches, fils ethernet, powerbars), comment ça fonctionnait l'année passée?

**Comment by Sytten** [comment]
On a des switchs dans le local (faudrait que je passe une fds pour les configurer). Sinon on emprunte les shits au (défunt) groupe viruslas. Il faut demander à l'AGEG pour la clé de la batcave (facepalm pour le nom) ou se trouve les trucs.
Ho j'ai pris la date de 2017, j'imagine que tu vas avoir la date de notre contact.

**Comment by Res260** [comment]
Viruslas était un groupe de l'AGEG?

**Comment by Res260** [comment]
Ouain jpense ça va être RIP pour le iHack :'(

**Comment by maximevaillancourt** [comment]
@Sytten 

Désolé pour le délai — oui, je confirme que la pizza du iHack Sherbrooke 2017 venait de chez Domino's Pizza. 🍕

**Comment by Sytten** [comment]
Est-ce qu'on a des nouvelles @Res260?

**Comment by maximevaillancourt** [comment]
@Res260 

Cédrick Chaput a été notre contact dans les dernières années. Si t'es sur le Slack de Hackfest (que tu peux rejoindre en remplissant [ce court formulaire](https://hackfestca.typeform.com/to/DW8455)), je pense que ça vaut la peine de le contacter pour savoir c'est quoi le statut.

**Comment by Res260** [comment]
@maximevaillancourt Je l'ai contacté, j'ai pas eu de réponse, @Sytten l'a poke sur Slack, il s'est fait dire qu'il n'organisait pas le iHack cette année par manque de temps.

**Comment by Res260** [comment]
RISING FROM THE ASHES

**Comment by Res260** [comment]
À jaser demain au diner: aider pour créer des challenges?

**Comment by Pourliver** [comment]
Comme mentionné, je serais intéressé (et émilio aussi messemble) à créer des challenges, mais ça dépend de la charge de travail. À confirmer avec eu et en discuter au prochain CE.

**Comment by Sytten** [comment]
Ouais surtout que personne ne semble prendre le lead. C'est un peu tard pour commencer, mais faudrait en parler avec le monde du hackfest.

**Comment by Res260** [comment]
Yeah j'en ai parlé à un dude, j'attend de ses nouvelles pour la date


--Émilio

**Comment by Res260** [comment]
7 juillet, j'ai contacté Marc-Henri.

**Comment by Sytten** [comment]
Parfait je vais m'occuper de l'organisation de la salle, si M-H répond, peux-tu me forward les mails? @Res260

**Comment by Res260** [comment]
Yeah will do.
